### DISCUSSION TITLE
Frigate LXC: Change video storage location via /etc/fstab or /etc/pve/lxe/{node}.conf?

### DISCUSSION THREAD

**Discussion by partytimeexcellent** [tteck/Proxmox#3877] [General]
I have my Frigate LXC installed on a small SSD. I want to store the videos and snapshots on a large local HDD (already initialized in PVE). What is the correct way to change the mapping? Do I edit the /etc/fstab in PVE, in the LXC, or do I edit the .conf file? Or is it as simple as adding the mount point via the GUI, or some combination?  Thanks!

**Comment by reubenb87** [comment]
Since my google came across your answer I'll share what I did. Mounted a NFS share in Proxmox host, then added as mount point in the priveledged LXC container, with storage as /media/frigate, then was automatically picked up after restart.

**Reply by Nikhilse120** [reply]
I have been trying to change the location of the recording using script lxc , can you help explain how do you change the location was it in the config.yml file ?
